user_management/api/views.py

```diff
@@ -59,6 +68,11 @@ def delete(self, request, *args, **kwargs):
 
 
 class UserRegister(generics.CreateAPIView):
+    """
+    User registration.
+
+    Register a user and send an email to validate the new account.
```

Comment:
I think the `User` can be set up to not require email validation. In this case `UserRegister